maropost.gemspec

```diff
@@ -11,30 +11,16 @@ Gem::Specification.new do |spec|
   spec.authors = [
```

Comment:
should we future proof the spec list here with red devs names + emails?

Reply:
oops, adding it